packages/react-core/src/components/TreeView/__tests__/__snapshots__/TreeView.test.tsx.snap

```diff
@@ -2309,11 +2309,11 @@ exports[`tree view renders icons successfully 1`] = `
                   height="1em"
                   role="img"
                   style="vertical-align: -0.125em;"
-                  viewBox="0 0 1217 1024"
+                  viewBox="0 0 576 512"
```

Comment:
When rebasing with the v5 branch, re-installing node modules and re-running tests, this snapshot was updated, which I would have expected to have been changed in a prior PR.